### PR TITLE
upgrade graphql v16.0.1

### PR DESCRIPTION
**Pull request by thedavidprice**
Release Notes:
- https://github.com/graphql/graphql-js/releases/tag/v16.0.0

Build Errors to fix:
- [x] packages/api/src/validations/validations.ts:567:32
- [x] packages/api/src/validations/validations.ts:568:25
- [x] packages/web/src/components/createCell.tsx:152:18
- [x] packages/forms/src/FormError.tsx:62:7

### CODE REVIEW

**Comment by thedavidprice** [comment]
Build Errors:
- packages/api/src/validations/validations.ts:567:32
- packages/api/src/validations/validations.ts:568:25
- packages/web/src/components/createCell.tsx:152:18
- packages/forms/src/FormError.tsx:62:7

**Review by dthyresson — COMMENTED** [review]
I have updated the PR to address the build issues. But @dac09 you may want to look at my creative TS.

**Comment by dac09** [comment]
TS looks fine to me DT, nicely done. Although I expect at somepoint we have to move form errors away from extensions!

**Comment by thedavidprice** [comment]
Updates look great @dthyresson Thank you! I'm waiting for CI to pass — assuming green I'll merge this in.

Question: 
Based on your changes, it doesn't look like there are any downstream breaking changes for Redwood projects due to this upgrade, correct?

Either way, I'll make note that this is a breaking change that could affect custom implementations. Just looking for additional specifics (in case there are any).

**Comment by dthyresson** [comment]
> TS looks fine to me DT, nicely done. Although I expect at somepoint we have to move form errors away from extensions!

Yes, the extensions are a leftover from Apollo Server and the client still likes them (unfortunately).

This is the item we discussed with Laurin from the Guild about union types and including errors as part of the SDL and not in extension. But that is a v2 possibility as is a huge change.

**Comment by thedavidprice** [comment]
E2E is hanging. Taking a look locally.

**Comment by thedavidprice** [comment]
@dthyresson Looks like the FormError change isn't working. Here's what I run into locally when trying the E2E "Getting Dynamic" — it throws when trying to save the new post:
```
TypeError: undefined is not an object (evaluating 'error.graphQLErrors[0].extensions['properties']')
...FormError.js
```

**Comment by dthyresson** [comment]
> @dthyresson Looks like the FormError change isn't working. Here's what I run into locally when trying the E2E "Getting Dynamic" — it throws when trying to save the new post:
> 
> ```
> TypeError: undefined is not an object (evaluating 'error.graphQLErrors[0].extensions['properties']')
> ...FormError.js
> ```

Ok I thought I ran e2e local but let me try again.

**Comment by dthyresson** [comment]
Ok so fixed ForError, but the change in the filter ops plugin here is failing. Hmm.

```
GraphQL operation type "mutation" is not allowed.
```

**Comment by thedavidprice** [comment]
locally its ✅

Waiting for CI E2E to officially give us the go ahead. Thanks!
